src/main/java/duke/other/DukeException.java

```diff
@@ -0,0 +1,19 @@
+package duke.other;
+
+/**
+ * Represents an Exception, specific to Duke. A DukeException object corresponds to an exception represented by a String
+ * error message.
+ */
+public class DukeException extends ArrayIndexOutOfBoundsException {
```

Comment:
Is there a reason why you chose to extend with `ArrayIndexOutOfBounds` over the base `Exception` class?